v1/core/tests/testtools/iothub_test/src/iothub_account.c

```diff
@@ -514,11 +522,10 @@ IOTHUB_ACCOUNT_INFO_HANDLE IoTHubAccount_Init_With_Config(IOTHUB_ACCOUNT_CONFIG*
 #else
                 iothub_account_info->connString = getenv("IOTHUB_CONNECTION_STRING");
                 iothub_account_info->eventhubConnString = getenv("IOTHUB_EVENTHUB_CONNECTION_STRING");
-                tempCert = getenv("IOTHUB_E2E_X509_CERT");
-                tempKey = getenv("IOTHUB_E2E_X509_PRIVATE_KEY");
+                base64_cert = getenv("IOTHUB_E2E_X509_CERT_BASE64");
```

Comment:
Will these new environment names require changes to the environment variables in Jenkins jobs?

Reply:
Yes, I should update the Jenkins job, but we are not using the cert in our tests.

Reply:
OK